proprietary/tokens/src/components/amsterdam/page-grid.tokens.json

```diff
@@ -1,20 +1,22 @@
 {
   "amsterdam": {
     "page-grid": {
-      "column-count": {
-        "narrow": {
-          "value": "4"
-        },
-        "wide": {
-          "value": "12"
-        }
+      "narrow": {
+        "column-count": { "value": "4" }
       },
-      "column-gap": {
-        "value": "clamp(0.5rem, calc(0.5rem + 0.0142857143 * (100vw - 20rem)), 1.5rem)"
+      "medium": {
+        "column-count": { "value": "8" }
       },
-      "max-width": {
-        "value": "1440px"
-      }
+      "wide": {
+        "column-count": { "value": "12" },
+        "max-width": { "value": "100rem" }
+      },
+      "x-wide": {
+        "column-count": { "value": "12" },
+        "max-width": { "value": "132rem" }
+      },
+      "column-gap": { "value": "clamp(0.5rem, 0.375rem + 3.125vw, 3.5rem)" },
+      "margin-column-width": { "value": "clamp(0.25rem, 0.1875rem + 1.625vw, 1.75rem)" }
```

Comment:
When we have everything working, I’ll try replacing these with their underlying values, e.g. `min-column-gap: 0.5rem, max-column-gap 3.5rem, margin-to-column-ratio: 1.5` so that the community can more easily apply their design choices rather than having to recalculate the `clamp` arguments.

Reply:
Reconsidered: leaving these clamped values in the `column-gap` and `margin-column-width` tokens improves reusability by allowing to override it with a fixed value.